### ISSUE TITLE
[Feature request] msbuild task for calling the generator will be nice to have

### ISSUE THREAD

**Issue by ArsenShnurkov** [macias/NaiveLanguageTools#3]
Build process is controlled by various tools like **msbuild**, **Cake**, **Nant** and so on. It will be nice to have an integration with some (or all) of them.

**Comment by macias** [comment]
Yes, probably, however it is another additional (extra) feature which I cannot execute right now. So thank you for the request but I have to leave it hanging.
